README.md

```diff
@@ -65,6 +84,21 @@ Replace `<something>` with the name of your S3 bucket, e.g. `wonderfulbali234`.
 }
 ```
 
+# CORS Policies
+
+A correctly configured bucket should have a [CORS policy](https://docs.aws.amazon.com/AmazonS3/latest/dev/cors.html) attached to it that permits requests from the domain where your Wiki is hosted to access the thumbnails uploaded by this extension. If the domain where your Wiki is hosted is at `www.example.com`, attach the following policy:
+
+```
+<CORSConfiguration>
+ <CORSRule>
+   <AllowedOrigin>http://www.example.com</AllowedOrigin>
+   <AllowedMethod>GET</AllowedMethod>
+ </CORSRule>
+</CORSConfiguration>
+```
+
+This addresses the issue when visitors to your Wiki receive the error message, "Sorry, the file cannot be displayed" when clicking on media.
```

Comment:
This is an internal error message of Extension:MultimediaViewer.
It shouldn't be mentioned as an example, as it never happens in MediaWiki without that extension.